networking_oneview/ml2/drivers/oneview/mech_oneview.py

```diff
@@ -34,6 +34,9 @@
 class OneViewDriver(api.MechanismDriver):
     def __init__(self):
         self.oneview_client = common.get_oneview_client()
+
+        common.check_valid_resources()
```

Comment:
we are checking before loading? is this ok?

Reply:
It is ok, we get the mapping directly from the conf file.

Reply:
It is ok, we load the mappings directly from the configuration file.